### PR TITLE
Deprecate `existingDataSeedInstance` option to CREATE DATABASE

### PR DESCRIPTION
**Pull request by mnd999**
Replace with new option `existingDataSeedServer`

Documenting https://github.com/neo-technology/neo4j/pull/27123

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1821.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1821.surge.sh/operations-manual/5/clustering/databases)
- [Create, start, and stop databases](https://neo4j-docs-operations-1821.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)

File: modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc [hunk 1]
```diff
@@ -76,13 +76,16 @@ The `CREATE DATABASE` command can have a map of options, e.g. `OPTIONS {key: 'va
 | `use`
 |
 Controls how the system handles existing data on disk when creating the database.
-Currently, this is only supported with `existingDataSeedInstance` and `seedURI`, and must be set to `use`, which indicates the existing data files should be used for the new database.
+Currently, this is only supported with `existingDataSeedInstance`, `existingDataSeedServer` and `seedURI`, and must be set to `use`, which indicates the existing data files should be used for the new database.
```
Comment (HannesSandberg) [inline modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc:79]: I can not make a comment above. But we should mention `existingDataSeedServer` on row 34 as well.

**Review by HannesSandberg — CHANGES_REQUESTED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1821.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1821.surge.sh/operations-manual/5/clustering/databases)
- [Create, start, and stop databases](https://neo4j-docs-operations-1821.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)

**Review by HannesSandberg — APPROVED** [review]

File: modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc [hunk 2]
```diff
@@ -76,13 +76,16 @@ The `CREATE DATABASE` command can have a map of options, e.g. `OPTIONS {key: 'va
...
-| `existingDataSeedInstance`
+
+|`existingDataSeedServer` label:new[new in 5.25]
+
+`existingDataSeedInstance` label:deprecated[deprecated in 5.25]
 | ID of the cluster server
 |
 Defines which server is used for seeding the data of the created database.
-The server ID can be found in the `serverId` column after running `SHOW SERVERS`.
+The server ID can be found in the `serverId` column after running `SHOW SERVERS`. `existingDataSeedInstance` was replaced by `existingDataSeedServer` in Neo4j 5.25.
```
Comment (renetapopova) [inline modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc:88]:
```suggestion
The server ID can be found in the `serverId` column after running `SHOW SERVERS`. 
`existingDataSeedInstance` is replaced by `existingDataSeedServer` in Neo4j 5.25.
```

File: modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc [hunk 3]
```diff
@@ -110,12 +113,12 @@ For details about enrichment mode, see link:{neo4j-docs-base-uri}/cdc/current/ge
 | `aligned` \| `standard` \| `high_limit` \| `block`
 |
 Defines the store format if the database created is new.
-If the store is seeded with `seedURI` or `existingDataSeedInstance`, or if the command is being used to mount pre-existing store files already present on the disk, they will use their current store format without any alterations.
+If the store is seeded with `seedURI`, `existingDataSeedInstance` or `existingDataSeedServer`, or if the command is being used to mount pre-existing store files already present on the disk, they will use their current store format without any alterations.
```
Comment (renetapopova) [inline modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc:116]:
```suggestion
If the store is seeded with `seedURI`, `existingDataSeedInstance` or `existingDataSeedServer`, or if the command is used to mount pre-existing store files already present on the disk, they will retain their current store format without any modifications.
```

**Review by renetapopova — APPROVED** [review]
Thanks, @mnd999. It looks good; I have just two minor suggestions.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1821.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1821.surge.sh/operations-manual/5/clustering/databases)
- [Create, start, and stop databases](https://neo4j-docs-operations-1821.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)

**Pull request merged**